projects/simplecore-ui/src/lib/card/card.ts

```diff
@@ -0,0 +1,47 @@
+﻿/**
+ * @license
+ * Copyright 2021 SimpleCore All Rights Reserved.
+ *
+ * Use of this source code is governed private licensing for
+ * internal SimpleCore projects
+ */
+
+import {
+  AfterContentInit,
+  AfterViewInit,
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  Input,
+  OnInit, Renderer2,
+  ViewChild,
+  ViewEncapsulation
+} from '@angular/core';
+
+@Component({
+  selector: 'core-card',
+  exportAs: 'coreCard',
+  template: '<div #cardEl class="core-card"><ng-content></ng-content></div>',
+  styleUrls: ['card.scss']
+})
+export class CoreCard implements AfterViewInit {
+  @Input() neumorphic: boolean = true;
+  @ViewChild('cardEl') cardEl: ElementRef;
+
+  constructor(private element: ElementRef, private renderer: Renderer2) {
```

Comment:
@officialsimplecore Please remove the unused dependency.  You do not use "element" in the code.

Reply:
Done